mlir/lib/Conversion/MIGraphXToTosa/MIGraphXToTosa.cpp

```diff
@@ -1226,9 +1226,15 @@ LogicalResult AsUnderlyingShapeConverter::matchAndRewrite(
   // in a standard shape. So, applying it to a logically-shaped tensor gets
   // you the tensor in in-memory layout.
   resultType.getStridePermutation(permutation);
+  // TOSA transpose takes i32
+  SmallVector<int32_t, 4> permutationI32;
+  permutationI32.reserve(permutation.size());
+  std::transform(permutation.begin(), permutation.end(), std::back_inserter(permutationI32),
```

Comment:
This might be `llvm::map_to_vector`

Reply:
thanks, I've changed it to llvm::map_to_vector